File: src/ERP/acme.atena.domain/DTO/Person/Fornecedor.cs
using acme.atena.domain.DTO.Account;
using acme.atena.domain.DTO.Product;
using acme.atena.domain.DTO.Util;
using System;
using System.Collections.Generic;
using System.Text;

namespace acme.atena.domain.DTO.Person
{
    public class Fornecedor : Pessoa
    {
        public Fornecedor()
        {
            Compras = new HashSet<Compra>();
            EnderecoFornecedores = new HashSet<EnderecoFornecedor>();
        }
        
        public string NomeFantasia { get; set; }
        public string InscricaoMunicipal { get; set; }

        public virtual ICollection<Compra> Compras{ get; set; }
        public virtual ICollection<EnderecoFornecedor> EnderecoFornecedores{ get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/person/fornecedor.ts
import { EnderecoFornecedor } from "../util/endereco-fornecedor";
import { Pessoa } from "./pessoa";

export class Fornecedor extends Pessoa{

  public nomeFantasia:string;
  public inscricaoMunicipal:string;

  public enderecoFornecedores: EnderecoFornecedor[];
}
